public/assets/cpu.js

```diff
@@ -76,6 +76,22 @@ function CPU(memamt) {
     var start = new Date();
     this.priv_reg[PCR["CSR_TIME"]["num"]] = Long.fromNumber(start.getTime());
 
+    // map of instruction types -> amounts
+    // this.instruction_amounts = new Map();
+    // this.instruction_amounts.set('arithmetic', 0);
+    // this.instruction_amounts.set('controlTransfer', 0);
+    // this.instruction_amounts.set('store', 0);
+    // this.instruction_amounts.set('load', 0);
+    // this.instruction_amounts.set('memoryOrder', 0);
+
+    this.instruction_amounts = {
```

Comment:
note, this doesn't *really* matter - i'm just letting you know: In JS, the convention is to use camel casing. Although I know some of the legacy code doesn't follow that convention.

Reply:
ah ok, I was just following their casing for their fields